api/src/main/resources/messages_fr.properties

```diff
@@ -236,3 +236,31 @@ isanteplusreports.number_of_confirmed_cases_for_severe_malaria_and_put_on_treatm
 isanteplusreports.number_of_patients_with_malaria_hospitalized=Nombre de patients avec Malaria hospitalis�s
 isanteplusreports.number_of_cases_of_confirmed_malaria_for_pregnant_women=Nombre de cas de la malaria confirm�e pour les femmes enceintes
 
+isanteplusreports.pnls.label=Rapport mensuel ART
+isanteplusreports.pnls.result.department.label=Dep't
+isanteplusreports.pnls.result.clinic.label=Clinique
+isanteplusreports.pnls.result.Males.label=M
+isanteplusreports.pnls.result.FeMales.label=F
+isanteplusreports.pnls.result.Subtotal.label=Sous-total
+isanteplusreports.pnls.result.BreastFeeding.label=Allaitement maternel
+isanteplusreports.pnls.result.1-0.label=0-1 Yrs
+isanteplusreports.pnls.result.1-4.label=1-4 Yrs
+isanteplusreports.pnls.result.5-9.label=5-9 Yrs
+isanteplusreports.pnls.result.10-14.label=10-14 Yrs
+isanteplusreports.pnls.result.15-19.label=15-19 Yrs
+isanteplusreports.pnls.result.20-24.label=20-24 Yrs
+isanteplusreports.pnls.result.25-29.label=25-29 Yrs
+isanteplusreports.pnls.result.30-34.label=30-34 Yrs
+isanteplusreports.pnls.result.35-39.label=35-39 Yrs
+isanteplusreports.pnls.result.40-44.label=40-44 Yrs
+isanteplusreports.pnls.result.45-49.label=45-49 Yrs
+isanteplusreports.pnls.result.50.label=50+ Yrs
+isanteplusreports.pnls.result.UnKnown.label=Inconnue Yrs
+isanteplusreports.pnls.result.Total.label=Total
+isanteplusreports.pnls.indicator.adult.label=Indicateurs du rapport mensuel ART
+isanteplusreports.pnls.result.pdf.header.label=Rapport mensuel ART
+isanteplusreports.pnls.result.pdf.creationDate.label=date cr��e
+isanteplusreports.newlyEnrolledPatientsOnArt=NOUVEAU 1.1: Nombre de s�ropositifs nouvellement inscrits au TAR pendant la p�riode s�lectionn�e (TX_NEW)
```

Comment:
I think the translation should be *"Nombre de personnes VIH+ nouvellement enrolées  sous ARV au cours du mois (TX_NEW)"* . Remove  *"NOUVEAU 1.1"*